gromacs/compat/__init__.py

```diff
@@ -0,0 +1,345 @@
+"""
+compat
```

Comment:
Yup, agreed - from a quick read one can indeed use `six` for this purpose. The `compat` library is ripped-out live from pandas, so I think they made the inverse decision: no external dependancy on six, let's just write down what we need.